### DISCUSSION TITLE
save selected object center

### DISCUSSION THREAD

**Discussion by Q210108** [opentoonz/opentoonz#6745] [Ideas]
![image](https://user-images.githubusercontent.com/100626161/158465068-52d17ab7-6722-4086-977d-14c41816a0a4.png)
i need this functionnality when i often rotate object or resize it because initial center is not always correct for our utilisation.

**Comment by RodneyBaker** [comment]
At one point a developer accidentally added that capability but it was fixed to make it go away.
It'd be useful to have that capability back.  :)

I will assume your primary interest here is with vectors as raster selections don't have the same capabilities for object groupings (as of yet).

**Comment by RodneyBaker** [comment]
Adding roadmap label at least temporarily as this is an oft cited requirement although in several parts.
There is the selected area's Center such as described here using the Selection Tool.
Then there is the Center that can be 'saved' but not animated which uses the Animate Tool.

At a minimum we should make sure the current usage is well documented and if we aren't going to adjust that should state such as well as provide workflow workarounds where possible.

**Comment by RodneyBaker** [comment]
Of note, the center of a selected object does get saved... while the current selection is active.
That selected center resets when a new selection is made.

This likely will not change unless/until a new process is introduced that creates/saves selection assignments for multiple selections which can then be reactivated by the user (with center data saved).

@Q210108 If you can share additional information regarding your desired workflow... perhaps a video of where the process fails as well as your requirement for the process to work correctly... that will be much appreciated.  

Added:  As referred to above this particular request likely relates to vector groups which could be made to store the center pivot data but currently do not.  If this is indeed the case renaming this report to 'Save Centers for Groups in Vector Levels' may be useful.  There is much that could be developed regarding vector groups and it would be useful to inventory those to better plan that development.

**Comment by RodneyBaker** [comment]
Moved this to Discussions due to the complexity related to development and the need for specific targets to hit.
These can only be determined through additional discussion.
